types/empower-core/index.d.ts

```diff
@@ -0,0 +1,83 @@
+// Type definitions for empower-core 1.2
+// Project: https://github.com/twada/power-assert-runtime
+// Definitions by: Jakub Jirutka <https://github.com/me>
```

Comment:
Please update the URL as well.

Reply:
Done

Reply:
Looks like it still says `https://github.com/me`, but should say `https://github.com/jirutka`.

Reply:
Oh, yes, I’m blind. :facepalm: